tests/Extensions/Diagnostics.HealthChecks.UnitTests/HttpHealthCheckParametersTests.cs

```diff
@@ -76,7 +76,7 @@ public void Constructor_InvalidSheme_ThrowException()
 		[TestMethod]
 		public void Constructor_AbsoluteUri_ThrowException()
 		{
-			Assert.ThrowsException<ArgumentException>(() => Create(relatedUri: new Uri("http://localhost")));
+			Assert.ThrowsException<ArgumentException>(() => Create(relatedUri: new Uri("https://localhost"), scheme: Uri.UriSchemeHttps));
```

Comment:
Do you need to add the scheme given that the URL itself specifies `https`?

Reply:
The method contract here is that it should receive relative url, so it won't have a scheme. This unit test checks that absolute url is an invalid parameter, but for some reason, I've wanted to keep other inputs totally valid, although it won't have any effect here.